libs/common/CMakeLists.txt

```diff
@@ -33,3 +33,36 @@ target_link_libraries(libs_timeout INTERFACE
   common
   rxcpp
   )
+
+add_library(irohad_version irohad_version.cpp)
+
+# Get the git repo data
+set(GIT_REPO_PRETTY_VER "version info unavailable")
+if (EXISTS "${PROJECT_SOURCE_DIR}/.git")
+  if(NOT GIT_FOUND)
+    find_package(Git QUIET)
```

Comment:
Seems, find_package will install git. I think better assume that git is already installed, but, ofc, there should be a fallback strategy and appropriate message when git is unavailable.

Reply:
We found out that the package will not be installed if missing.